src/components/RoomDropdown/index.tsx

```diff
@@ -0,0 +1,159 @@
+import React, { useEffect, useRef } from 'react'
+import { useState } from 'react'
+import styled from 'styled-components'
+import {
+    AddMemberIcon,
+    CopyIcon,
+    DeleteIcon,
+    DownloadIcon,
+    EditMembersIcon,
+    SearchIcon,
+} from '../../icons'
+import { KARELIA } from '../../utils/css'
+
+type Props = {
+    button?: any
```

Comment:
We could
1. type it better (e.g. `ReactNode`, for starters), but we could also
2. bake `RoomAction` into the `RoomDropdown` and don't bother with a custom props-driven thing.

Let's keep it simple.

Plus, the `RoomDropdown` could land inside `pages/Chat` cause we may never use it anywhere else.

Let's make things reusable when the time comes. ;)

Reply:
I changed it to `React.ReactElement` since the `React.cloneElement` requires `ReactElement`.